debug/windows/npm/win-debug.ps1

```diff
@@ -0,0 +1,28 @@
+$filepath = "logs_$((Get-Date).ToString('MM-dd-yyyy'))"
+kubectl get pod -A -o wide >> ( New-Item -Path ./$filepath/allpods.out -Force )
+$npmpod = kubectl get pod -n kube-system -owide --output=custom-columns='Name:.metadata.name,Node:spec.nodeName' | Select-String "npm-win"
+$rows = @()
+foreach ($row in (-split $npmpod)) {
+    $rows += $row
+}
+
+for ($i = 0; $i -lt $rows.Length; $i += 2) {

```

Comment:
can you add some prints to add some info on what it is collecting, where it is storing etc , so that the script does not feel stuck

Reply:
added